lib/api/bucketPut.js

```diff
@@ -37,10 +39,25 @@ export default function bucketPut(authInfo, request, locationConstraint, log,
         log.trace('invalid acl header');
         return callback(errors.InvalidArgument);
     }
+    // - AWS JS SDK sends a request with locationConstraint us-east-1
+    // if no locationConstraint provided.
```

Comment:
Nevermind, I misunderstood what the comment was saying. Is the comment still relevant to this piece of code?